### PR TITLE
simple style guide added to appendix

### PR DESCRIPTION
**Pull request by welzie**

### CODE REVIEW

File: src/appendices/style-guide.rst [hunk 1]
```diff
@@ -0,0 +1,121 @@
...
+   if (fuelTempCelsius > -220) 
+   {
+      console.log('WARNING');
+   } 
+   else 
+   {
+      console.log('Temp fine');
+   }
+
+Always use Braces for If Statements
```
Comment (chrisbay) [inline src/appendices/style-guide.rst]: This also applies to loops.

File: src/appendices/style-guide.rst [hunk 2]
```diff
@@ -0,0 +1,121 @@
...
+      console.log('WARNING');
+
+Use Semicolons
+^^^^^^^^^^^^^^
+
+**Good**
+
+.. sourcecode:: js
+
+   let ir = false;
```
Comment (chrisbay) [inline src/appendices/style-guide.rst]: We probably shouldn't use a variable name we previously called "bad" in a later section labeling the code example "good."

File: src/appendices/style-guide.rst [hunk 3]
```diff
@@ -0,0 +1,121 @@
...
+
+**Bad**
+
+.. sourcecode:: js
+
+   const c = 7;
+   const fuelTempC = -225;
+   let ir = false;
+
+Opening Braces and Statements on Same Line
```
Comment (chrisbay) [inline src/appendices/style-guide.rst]: It strikes me that there is nothing in the guide about indentation. I'm less concerned with the tab-space argument than that conditional and loop bodies should be indented by one "tab."

File: src/appendices/style-guide.rst [hunk 4]
```diff
@@ -0,0 +1,121 @@
...
+When writing code there are the specific language rules you must follow for your code to execute. In addition to those language rules
+there are **style guides** which define style rules to be followed when writing code.
+
+There are multiple style guides for each language. For consistency sake, it's important to follow the style guide rules of your organization
+or company when writing code. The style guide for this course is below.
+
+JavaScript Style Guide
+----------------------
+
+Camel Case Variable Names
```
[the remark this answers is not in the document]
Comment (welzie) [reply src/appendices/style-guide.rst]: I prefer camel case. Do we fight it out or ask a wider audience?

[the remark this answers is not in the document]
Comment (chrisbay) [reply src/appendices/style-guide.rst]: I'll change existing references to use the phrase "camel case," with a brief note about "lower camel case."

File: src/appendices/style-guide.rst [hunk 5]
```diff
@@ -0,0 +1,121 @@
+Style Guide
+===========
+
+When writing code there are the specific language rules you must follow for your code to execute. In addition to those language rules
```
[the remark this answers is not in the document]
Comment (chrisbay) [reply src/appendices/style-guide.rst]: I still feel like this lacks something. Can we mention that following a style guide enhances readability for yourself and other programmers? Even a motivating example could help: "Imagine a book written by multiple authors, each of which uses different practices for capitalization, indentation, and punctuation. One writes in all-lowercase and never indent paragraphs. Another capitalizes every word and uses only the period for punctuation." Maybe this is too much, but conveying the importance of readability seems important.

File: src/appendices/style-guide.rst [hunk 6]
```diff
@@ -0,0 +1,121 @@
...
+there are **style guides** which define style rules to be followed when writing code.
+
+There are multiple style guides for each language. For consistency sake, it's important to follow the style guide rules of your organization
```
Comment (chrisbay) [inline src/appendices/style-guide.rst]: "For the sake of consistency..."

**Pull request merged**